modules/test/playwright/tests/client-extension-web/pages/EditClientExtensionsPage.ts

```diff
@@ -33,6 +34,10 @@ export class EditClientExtensionsPage {
 			name: 'Publish',
 		});
 
+		this.descriptionCKEditor = page.locator(
+			'#cke__com_liferay_client_extension_web_internal_portlet_ClientExtensionAdminPortlet_description'
+		);
+
```

Comment:
As spoken this morning, this is kind of fixing the symptom and not the root cause of elements not being available. We will tackle this in depth [here](https://liferay.atlassian.net/browse/LPD-25816)